lib/rules/max-line-length/index.js

```diff
@@ -51,6 +46,11 @@ const rule = (primary, secondaryOptions, context) => {
 			throw new Error('The root node must have a source');
 		}
 
+		const EXCLUDED_PATTERNS = [
+			/url\(\s*(\S.*\S)\s*\)/gi, // allow tab, whitespace in url content
+			/@import\s+(['"].*['"])/gi,
+		];
+
```

Comment:
[note] Bring the definition and the place of use closer together for readability.